custom_components/smartthinq_sensors/sensor.py

```diff
@@ -589,3 +644,20 @@ def extra_state_attributes(self):
         data.update(features)
 
         return data
+
+
+class LGEAirPurifierSensor(LGESensor):
+    """A sensor to monitor LGE air purifier devices"""
+
+    def __init__(
+            self,
+            device: LGEDevice,
+            description: ThinQSensorEntityDescription,
+    ):
+        """Initialize the sensor."""
+        super().__init__(device, description, LGEAirPurifierDevice(device))
+
+    @property
+    def extra_state_attributes(self):
+        """Return the optional state attributes."""
+        return None
```

Comment:
This class is useless if you don't want to expose specific attributes. Just remove the class and use `LGESensor(lge_device, sensor_desc, LGEAirPurifierDevice(device))` in `async_setup`

Reply:
Why this class is back here?

Reply:
No problem, I merge this PR and fix in release.